back-end/.env.dist

```
@@ -7,4 +7,7 @@ JWT_EXPIRATION_TIME=999999999
 
 # Client
 FRONTEND_DOMAIN=localhost
-FRONTEND_URL=http://localhost:3001
\ No newline at end of file
+FRONTEND_URL=http://localhost:3001
+
+# Server
+PORT=3000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)